azure-cli2017.pyproj

```diff
@@ -13,8 +13,7 @@
     <LaunchProvider>Standard Python launcher</LaunchProvider>
     <InterpreterId>MSBuild|env|$(MSBuildProjectFullPath)</InterpreterId>
     <EnableNativeCodeDebugging>False</EnableNativeCodeDebugging>
-    <CommandLineArguments>
-    </CommandLineArguments>
+    <CommandLineArguments>webapp log config --name ps9085 --resource-group ps6148 --application-logging true</CommandLineArguments>
```

Comment:
this should not be checked in to code

Reply:
Furthermore, I doubt that we should put .pyproj file into git. It is a file designed for certain IDE.